src/routes/concepts/stores.mdx

```diff
@@ -87,14 +89,75 @@ const App = () => {
 }
 ```
 
+When you create a store, it contains the initial state, but _no signals_ are immediately created to track changes to that state.
+These signals are created lazily, meaning they come into existence only when you access the data within a reactive context.
+
+When you use this data within a reactive context, such as in a template, computed property, or effect, that is when the signals are created and dependencies are established.
+
+For example, if you wanted to you wanted to print out every new user, adding the 
+line below won't work because it is not within a tracked scope.
```

Comment:
```suggestion
Once data is used within a reactive context, such as within the return statement of a function, computed property, or an effect, a signal is created and dependencies are established.

For example, if you wanted to print out every new user, adding the console log below will not work because it is not within a tracked scope.
```

Reply:
@LadyBluenotes I agree that my use of "template" was not specific enough, but I think it needs to be "within the return statement of a **component** function". Just saying function will probably give the wrong impression that any functions return statement is tracked.

```suggestion
Once data is used within a reactive context, such as within the return statement of a component function, computed property, or an effect, a signal is created and dependencies are established.
	
For example, if you wanted to print out every new user, adding the console log below will not work because it is not within a tracked scope.
```